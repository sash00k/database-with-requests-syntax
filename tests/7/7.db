{"name" : "Иванов А. С.", "birthday" : "10.01.2003", "id" : 1522347612, "department" : "ММФ", "group" : 22129, "student_id" : 421254, "course" : "Ан.геом", "groups" : [22129], "students" : [12455, 1251224]}
{"name" : "Петров А. С.", "birthday" : "15.11.2003", "id" : 12545561, "department" : "ММФ", "group" : 22129, "student_id" : 12455, "course" : "Матан", "groups" : [22129], "students" : [421254, 1251224]}
{"name" : "Сидорова А. С.", "birthday" : "12.02.2003", "id" : 77341241, "department" : "ММФ", "group" : 22129, "student_id" : 1251224, "course" : "Алгебра", "groups" : [22129], "students" : [12455, 421254]}