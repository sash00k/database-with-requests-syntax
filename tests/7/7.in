get records where students is set