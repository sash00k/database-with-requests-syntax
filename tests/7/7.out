Student Иванов А. С. from group 22129, ММФ. Also teaches Ан.геом in 1 groups and mentors 2 students.
Student Петров А. С. from group 22129, ММФ. Also teaches Матан in 1 groups and mentors 2 students.
Student Сидорова А. С. from group 22129, ММФ. Also teaches Алгебра in 1 groups and mentors 2 students.